src/test/resources/outputs/redshift/builtins/abs.sql

```diff
@@ -0,0 +1,80 @@
+-- NOTE: Redshift does not support top-level expression syntax.
```

Comment:
Double checking, for features that are not supported in the target dialogue, does the test framework assert on exception?